app/webpacker/styles/lead_providers/syntax_highlighting.scss

```diff
@@ -0,0 +1,226 @@
+// A Base16 palette
+// https://github.com/chriskempson/base16
+
+$code-00: govuk-colour("light-grey"); // Default Background
+$code-01: #f5f5f5; // Lighter Background (Unused)
+$code-02: #bfc1c3; // Selection Background
+$code-03: darken( $govuk-secondary-text-colour, 2%); // Comments, Invisibles, Line Highlighting
+$code-04: #e8e8e8; // Dark Foreground (Unused)
```

Comment:
If we are copying this from another project, we should put them in `vendor_styles`, preferably with a comment saying where we got them from so we can update them easily.

Reply:
copied from Apply, but just a custom CSS style for https://github.com/rouge-ruby/rouge based on http://pygments.org

Reply:
What you just wrote should be in a code comment, so some poor soul who looks at it in a few months knows where to look for the original.

And please move it to `app/webpacker/styles/vendor`, like we do with `nhs_table` component.

Reply:
added source comments where applicable